app/api/currentuser/get_group_activities.go

```diff
@@ -92,8 +92,10 @@ type rootItem struct {
 // description:
 //   If `{watched_group_id}` is not given, the service returns the list of root activities of the groups the current user
 //   (or `{as_team_id}`) belongs to.
-//   Otherwise, the service returns the list of root activities of all ancestor groups of the watched group which are also
+//   Otherwise, the service returns the list of root activities (visible to the current user or `{as_team_id}`)
+//   of all ancestor groups of the watched group which are also
 //   ancestors or descendants of at least one group that the current user manages.
```

Comment:
Reading  "of all ancestor groups of the watched group which are also ancestors or descendants of at least one group that the current user manages." -> but as the watched group is manage, all ancestors of the watched group are "ancestors or descendants of at least one group that the current user manages". Am I wrong?

Reply:
Not really.
```
A   B
  \ /
   C
```
If the current user is a manager of B, and the watched group is C, then the current user can manage C, but the root activity of A will not be displayed although it is an ancestor of the watched group.

Reply:
You should use "explicitly manage" and "implicitly manage". 
So your "at least one group that the current user manages" means "explicitly manage" while "the user should ba a manager of the group" two sentences below mean "implicitly manage"?

Reply:
Fixed in #744